src/actions/index.js

```diff
@@ -3,12 +3,23 @@ import * as types from './types'
 import { api } from 'apis'
 
 export const fetchUser = () => async dispatch => {
-  dispatch({ type: types.USER_REQUEST })
+  dispatch({ type: types.FETCH_USER_REQUEST })
```

Comment:
adjusting for VERB_ENTITY_STATUS naming convention

Reply:
.---.